crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/integration/CrownCourtProceedingIntegrationTest.java

```diff
@@ -103,7 +102,8 @@ void givenValidContent_whenApiResponseIsError_thenProcessRepOrderIsFails() throw
                         HttpMethod.POST, objectMapper.writeValueAsString(apiProcessRepOrderRequest), ENDPOINT_URL))
                 .andExpect(status().is5xxServerError())
                 .andExpect(content().contentType(MediaType.APPLICATION_JSON))
-                .andExpect(jsonPath("$.message").value("500 Internal Server Error from GET " + stubUrl));
+                .andExpect(jsonPath("$.message", containsString("500 Internal Server Error")))
+                .andExpect(jsonPath("$.message", containsString("/ioj-appeal/repId/" + TEST_REP_ID + "/current-passed")));
```

Comment:
I realised I showed it like this in the example, but no reason not to use the `stubPath` again here.